Marlin/ultralcd.cpp

```diff
@@ -1683,12 +1683,18 @@ void kill_screen(const char* lcd_msg) {
      * If the queue is full, the command will fail, so we have to loop
      * with idle() to make sure the command has been enqueued.
      */
-    void lcd_enqueue_command_sram(char * const cmd) {
+    void lcd_enqueue_command(char * const cmd) {
       no_reentry = true;
       while (enqueue_and_echo_command(cmd)) idle();
       no_reentry = false;
     }
 
+    void lcd_enqueue_commands_P(const char * const cmd) {
+      no_reentry = true;
+      while (enqueue_and_echo_commands_P(cmd)) idle();
```

Comment:
Because of the way that `drain_injected_commands_P` and hence `enqueue_and_echo_commands_P` behaves, this could be very bad. To do what you're attempting, you should instead…
```cpp
enqueue_and_echo_commands_P(cmd);
while (drain_injected_commands_P()) idle();
```